LogParser.cs

```diff
@@ -150,76 +186,87 @@ private string getLine(byte[] data, int offset)
             owner.LineCount++;
 
             //logger.Debug("End Method: LogParser.getLine(), returning " + s);
-			return s;
-		}
+            return s;
+        }
 
-		public void OnChanged(object source, FileSystemEventArgs e)
-		{
+        public void OnChanged(object source, FileSystemEventArgs e)
+        {
             logger.Debug("Begin Method: OnChanged(object,FileSystemEventArgs) (" 
                 + source.ToString() + "," + e.ToString() + ")");
 
             changed = true;
 
             logger.Debug("End Method: OnChanged()");
-		}
+        }
 
-		public LogLineType Parse(string s)
-		{
-			if (TellsOn)
-			{
+        public LogLineType Parse(string s)
+        {
+            if (TellsOn)
+            {
                 Regex r = new Regex(@"\[.*\] (?<name>\S+) (tells|told) you, '(?<message>.*)'");
-				Match m = r.Match(s);
+                Match m = r.Match(s);
 
-				if (m.Success) 
-				{
+                if (m.Success) 
+                {
                     logger.Debug(s + " matches tell regex.");
 
                     owner.ParseCount++;
 
                     string name = m.Groups["name"].ToString();
-					if (!Tells.Contains(name)) 
-					{
+                    if (!Tells.Contains(name)) 
+                    {
                         logger.Debug(name + " is not in tell array already.");
 
-						Tells.Add(name);
-						DataRow[] k = namesTiers.Select("Name='" + name + "'");
-						if (k.Length>0) 
-						{
-							TellsDKP.Add(new Raider(owner,(string)k[0]["Name"],(string)k[0]["Tier"],(double)k[0]["SumOfPTS"],(double)Decimal.ToDouble((decimal)k[0]["TPercent"])));
-						} 
-						else 
-						{
-							TellsDKP.Add(new Raider(owner,name,Raider.NOTIER,Raider.NODKP,Raider.NOATTENDANCE));
-						}
-						TellsDKP.Sort();
+                        Tells.Add(name);

```

Comment:
Below here are the changes that matter for populating the tell table, thorugh line 241.